app/modules/bme280.c

```diff
@@ -144,24 +148,29 @@ static uint8_t w8u(uint8_t reg, uint8_t val) {
 	platform_i2c_send_stop(bme280_i2c_id);
 }
 
-static uint16_t r16u(uint8_t reg) {
-	uint8_t high = r8u(reg);
-	uint8_t low  = r8u(++reg);
-	return (high << 8) | low;
+static uint8_t r8u(uint8_t reg) {
+	uint8_t ret[1];
+	r8u_n(reg, 1, ret);
+	return ret[0];
 }
 
-static uint16_t r16uLE(uint8_t reg) {
-	uint8_t low = r8u(reg);
-	uint8_t high  = r8u(++reg);
-	return (high << 8) | low;
-}
+// static uint16_t r16u(uint8_t reg) {
```

Comment:
Please either put these functions into a meaningful `#ifdef XYZ` / `#endif` or remove them completely if not used anymore.